content/en/faq/metrics-and-logs/mixer-migration.md

```diff
@@ -0,0 +1,16 @@
+---
+title: How to migrate from Mixer based extensbility?
+weight: 30
+---
+
+Mixer has been [removed from Istio since 1.8 release](/news/releases/1.8.x/announcing-1.8/#deprecations).
+Migration is needed if you are still rely on Mixer built-in adapter or out-of-process adapter for mesh extension.
+
+For built-in adapters, several alternatives are provided:
+
+* `Prometheus` and `Stackdriver` adapters are replaced by telemetry v2 and implemented as [proxy extensions](/docs/reference/config/proxy_extensions/).
+  Customization of telemetry generated by these two extensions can be achieved via [request classification](/docs/tasks/observability/metrics/classify-metrics/).
+* `Memquota` and `Redisquota` adapters are replaced by the [Envoy based rate limiting solution](/docs/tasks/policy-enforcement/rate-limit/).
+* `OPA` adapter is replaced by the [Envoy ext-authz based solution](/docs/tasks/security/authorization/authz-custom/), which supports [integration with OPA policy agent](https://www.openpolicyagent.org/docs/latest/envoy-authorization/).
+
+For out-of-process adapter, Wasm based extension can be used. Please refer to the guides about [Wasm module development](https://github.com/istio-ecosystem/wasm-extensions/blob/master/doc/write-a-wasm-extension-with-cpp.md) and [distribution](/docs/ops/configuration/extensibility/wasm-module-distribution/).
```

Comment:
Should we also point users to the "how to run mixer with newer Istio releases" wiki doc you made earlier ?

Reply:
Added. I also wondered whether we should add this item here or not when created this PR, and left it off because of maintenance concern. I added it with caveat, so that user would not have high expectation about this temporary solution.